custom-connectors/AzureOpenAIService/apiDefinition.swagger.json

```diff
@@ -0,0 +1,657 @@
+                },
+                "object": {
+                  "type": "string",
+                  "title": "Object",
+                  "description": "The type of object."
+                }
+              }
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ]
+      },
+      "post": {
+        "summary": "Create deployment",
+        "operationId": "CreateDeployment",
+        "description": "Creates a new deployment for the Azure OpenAI resource according to the given specification.",
+        "x-ms-visibility": "advanced",
+        "responses": {
+          "201": {
+            "$ref": "#/responses/createdeployment201"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "model": {
+                  "type": "string",
+                  "description": "The Azure OpenAI Service model identifier (model-id) to deploy. Can be a base model or a fine tune.",
+                  "title": "Model",
+                  "x-ms-visibility": "important",
+                  "x-ms-dynamic-values": {
+                    "operationId": "ListModels",
+                    "value-path": "id",
+                    "value-collection": "data",
+                    "value-title": "id",
+                    "parameters": {
+                      "api-version": "2022-12-01"
+                    }
+                  }
+                },
+                "scale_settings": {
+                  "$ref": "#/definitions/scale_settings"
+                }
+              },
+              "required": [
+                "scale_settings",
+                "model"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/openai/deployments/{deployment-id}": {
+      "get": {
+        "summary": "Get deployment",
+        "operationId": "GetDeployment",
+        "description": "Gets details for a single deployment specified by the given deployment.",
+        "x-ms-visibility": "important",
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "$ref": "#/definitions/models"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/deployment"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ]
+      },
+      "delete": {
+        "summary": "Delete deployment",
+        "description": "Deletes the specified deployment.",
+        "operationId": "DeleteDeployment",
+        "x-ms-visibility": "advanced",
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}

```

Comment:
@Laskewitz empty schema is not allowed, can you please define a proper schema for the response.
Please fix all empty schemas in the swagger file

Reply:
Okay we will accept this as a custom connector. please be aware that this is not allowed for certified connectors.
Thank you so much again.